src/coprocessor/dag/executor/scanner.rs

```diff
@@ -121,7 +121,7 @@ impl<S: Snapshot> Scanner<S> {
         }
 
         self.seek_key = match (self.scan_mode, self.scan_on) {
-            (ScanMode::Forward, _) => util::prefix_next(&key),
+            (ScanMode::Forward, _) => util::prefix_next(&key), // Where is this thing useful?????
```

Comment:
I think we don't need to calculate seek_key any more in forward scan? Also you may take a look at this for some background: https://github.com/pingcap/tikv/issues/3157